modules/renter/proto/editor.go

```diff
@@ -255,12 +265,22 @@ func (he *Editor) Modify(oldRoot, newRoot crypto.Hash, offset uint64, newData []
 
 // NewEditor initiates the contract revision process with a host, and returns
 // an Editor.
-func NewEditor(host modules.HostDBEntry, contract modules.RenterContract, currentHeight types.BlockHeight, cancel <-chan struct{}) (*Editor, error) {
+func NewEditor(host modules.HostDBEntry, contract modules.RenterContract, currentHeight types.BlockHeight, hdb hostDB, cancel <-chan struct{}) (_ *Editor, err error) {
 	// check that contract has enough value to support an upload
 	if len(contract.LastRevision.NewValidProofOutputs) != 2 {
 		return nil, errors.New("invalid contract")
 	}
 
+	// Increase Successful/Failed interactions accordingly
+	defer func() {
+		// a revision mismatch is not necessarily the host's fault
+		if err != nil && !IsRevisionMismatch(err) {
```

Comment:
hmm. A revision mismatch can be the host's fault too though. Generally speaking the renter is constructed so that as long as the host is honest, there will not be a revision mismatch. It's probably fine to count a mismatch as a failed interaction